etc/scripts/changelog.py

```diff
@@ -48,18 +45,17 @@ def html_escape(text):
 
 
 def format_commit(url, commit):
-    grouped_commit = commit.split(' * ')
-    # --grep matches every line on the commit message, we ensure we don't get caught by it
+    # Input format is "<hash> <description>", so split only on the first space and escape the commit message
+    grouped_commit = commit.split(' ', 1)
     try:
-        formatted_commit = commit_template.format(html_escape(grouped_commit[1]), url, grouped_commit[0])
-        print(grouped_commit)
```

Comment:
Please leave this print so that changelog entries get logged as part of the CI build

Reply:
Should be done! Moved it up since it isn't related to the try-catch.